cli/build.rs

```diff
@@ -127,10 +125,10 @@ mod ts {
       build_libs: Vec<&'static str>,
       path_dts: PathBuf,
     },
-    state = |state, op_crate_libs, build_libs, path_dts| {
-      state.put(op_crate_libs);
-      state.put(build_libs);
-      state.put(path_dts);
+    state = |state, cfg| {
```

Comment:
I know this will be a nitpick, but we mostly used term "options" for these kind of things - eg. `deno_fetch::Options`. Any chance we could stick to that?

Reply:
Oh yeah, I can switch that out.